### ISSUE TITLE
Editor/Resources items are built into the player, they shouldn't be

### ISSUE THREAD

**Issue by doctorpangloss** [inkle/ink-unity-integration#80]
According to Build Report Tool. I can confirm the size of the build increases with the images. I know it's just about 0.5 MB but I'm making a WebGL game carefully tuned to fit into 7.0 MB!

This might be a longstanding Unity issue. The workaround is to delete the files and commenting out the EditorGUI calls. Moving the files to Editor Default Resources also fixes the problem but creates another: the editor code needs to be updated to find the files there.

**Comment by tomkail** [comment]
Yeah we should resolve this. I've not got the time right now, but if anyone has time to resolve this I'll take the commit :) thanks!

**Comment by tomkail** [comment]
Hum so you cant use Editor Default Resources because that folder is unique, and must be in the root. Bit unsure what the correct way to go about this is! 
We need to load a texture into a static class without using Resource/EditorResource Load

**Comment by tomkail** [comment]
Fixed!
